addons/controllers/antrea/antreaconfig_util.go

```diff
@@ -35,42 +42,65 @@ type antrea struct {
 }
 
 type antreaNsx struct {
-	Enable          bool                   `yaml:"enable,omitempty"`
-	BootstrapFrom   antreaNsxBootstrapFrom `yaml:"bootstrapFrom,omitempty"`
-	AntreaNsxConfig antreaNsxConfig        `yaml:"config,omitempty"`
+	Enable          bool            `yaml:"enable,omitempty"`
+	AntreaNsxConfig antreaNsxConfig `yaml:"config,omitempty"`
```

Comment:
Looks like AntreaNsxConfig should be moved to AntreaConfigSpec, and `yaml:"config,omitempty"` should be changed to `yaml:"antrea_interworking,omitempty"`. This is to be aligned with Antrea package schema.yaml.

Reply:
done.